website/source/docs/secrets/mssql/index.html.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: "docs"
+page_title: "Secret Backend: mssql"
+sidebar_current: "docs-secrets-mssql"
```

Comment:
There should be an entry in `website/source/layouts/docs.erb` for `docs-secrets-mssql` for this to work.